configure.ac

```
@@ -48,12 +48,12 @@ AC_PREFIX_DEFAULT([/usr])
 
 dnl multiple personality support (x86 & x86_64: multilib)
 AC_MSG_CHECKING([for multiple personalities])
-AC_ARG_ENABLE([schizo],
-	[AS_HELP_STRING([--enable-schizo],[Support multiple personalities])],
-	[],[enable_schizo="auto"])
-AC_MSG_RESULT([$enable_schizo])
-SB_SCHIZO_SETTINGS=
-AC_DEFUN([SB_CHECK_SCHIZO],[dnl
+AC_ARG_ENABLE([personalities],
+	[AS_HELP_STRING([--enable-personalities],[Support multiple personalities])],
```

It's definitely an improvement, but it still might not be obvious what "personalities" means in this context. It would be nice to provide a more self-evident explanation, but I can't think of anything succinct.

Maybe add the words "using ptrace", since this option only effects that part of the sandbox code.

---

Taking into account your suggestion, I think adding "Linux" might help too, given it's got a specific meaning there (and I assume that's part of it here):
```suggestion
	[AS_HELP_STRING([--enable-personalities],[Support multiple Linux personalities using ptrace])],
```